src/plugins/home/public/application/components/guided_onboarding/getting_started.tsx

```diff
@@ -36,38 +54,37 @@ const title = i18n.translate('home.guidedOnboarding.gettingStarted.useCaseSelect
   defaultMessage: 'What would you like to do first?',
 });
 const subtitle = i18n.translate('home.guidedOnboarding.gettingStarted.useCaseSelectionSubtitle', {
-  defaultMessage: `Select a guide to help you make the most of your data.`,
+  defaultMessage: `Filter by solution to see related use cases`,
 });
 const skipText = i18n.translate('home.guidedOnboarding.gettingStarted.skip.buttonLabel', {
   defaultMessage: `I’d like to explore on my own.`,
 });
 
 export const GettingStarted = () => {
-  const { application, trackUiMetric, chrome, guidedOnboardingService, cloud } = getServices();
+  const {
+    application,
+    trackUiMetric,
+    chrome,
+    guidedOnboardingService,
+    cloud,
+    openModal,
+    theme,
+    i18nStart,
+    docLinks,
+    share,
+  } = getServices();
 
   const [guidesState, setGuidesState] = useState<GuideState[]>([]);
   const [isLoading, setIsLoading] = useState<boolean>(false);
   const [isError, setIsError] = useState<boolean>(false);
+  const [filteredCards, setFilteredCards] = useState<GuideCardConstants[]>();
   const { search } = useLocation();
   const query = parse(search);
-
-  const isTypeOfGuideFilterValue = (useCase: string | string[] | null) => {
-    const filterValues: string[] = ['search', 'observability', 'security', 'all']; // list of GuideFilterValues types
-
-    if (!useCase) {
-      return false;
-    }
-
-    if (useCase instanceof Array) {
-      return filterValues.includes(useCase[0]);
-    }
-
-    return filterValues.includes(useCase);
-  };
-
-  const [filter, setFilter] = useState<GuideFilterValues>(
-    isTypeOfGuideFilterValue(query.useCase) ? (query.useCase as GuideFilterValues) : 'all'
+  const [filter, setFilter] = useState<GuideFilterValues | GuideFilterValuesClassic>(
+    (query.useCase as GuideFilterValues) ?? 'search'
   );
+  // using for A/B testing
+  const [classicGuide] = useState<boolean>(false);
```

Comment:
Are you planning to follow up on this in a separate PR?

Reply:
Yes! Hoping to get this initial push in and hopefully add to it before FF or worse case integrate with Launch Darkly after FF.